### DISCUSSION TITLE
Bulk Updater to save newly created Bi-Directional Related Custom Fields

### DISCUSSION THREAD

**Discussion by cjc1867** [pods-framework/pods#6705] [Ideas]
Hi
I have added 334 records and my related field to add the Clergy's name wasn't bi-driectional so now I have to open each record and re-save it before it can recognise the Clergy's name.

Apart from being time consuming, surely there should be an easier way of doing this with code.

I won't be the only person who has done this, so how have they dealt with this problem?

There should be a utility/add-on that can handle this problem. No doubt, I will hit this problem with my other Pods as well.

Colin

**Comment by JoryHogeveen** [comment]
Hi @cjc1867 

Please fill in a complete bug report for this.
Also include steps for us to reproduce your issue etc.

Cheers, Jory

**Comment by cjc1867** [comment]
Where do I complete a full bug report?

Just create a one way link between 2 Pods and add 300+ records, then go back and change the link to bi directional or 2 way link and only one side will have the records. The other will not have the related records as you now have to open each original record and resave it.

Only then will both sides have related records.

After searching for an answer, I found that other people had the same issue and I couldn't find a definite answer that worked. This is an issue that needs fixing.

Colin

**Comment by JoryHogeveen** [comment]
Hi @cjc1867 

Ah check, I understand what you mean. Once a field is changed the field data indeed isn't auto-updated.
This will indeed require a re-save of all your posts.

One way you could do this is by using the WordPress's core feature "Bulk edit".

I also found two topics about this that might help doing the same thing by code:
https://wordpress.stackexchange.com/questions/330746/how-can-i-mass-update-save-all-wordpress-posts-and-pages/330751
https://wordpress.stackexchange.com/questions/294575/how-to-force-update-all-posts-after-import

Cheers, Jory

**Comment by cjc1867** [comment]
Hi Jory
Bulk Edit does not work as I have tried it.

Colin

**Comment by cjc1867** [comment]
I want to add the related field to the quick edit or bulk edit and then see if that works.

Should be able to do this via a plugin or functions.php.

Also we should be saving the post meta as well as the post in quick edit.

Not sure how to do it though.
